src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,9 +40,7 @@ object Homework :
 
   object `Boolean Operators` :
 
-    val int = 42
-
-    def not(b: Boolean): Boolean = ??? // here is my greatest solution
+    def not(b: Boolean): Boolean = !b // here is my greatest solution
```

Comment:
Please fix `not`, it's not allowed to use boolean negation, use axioms for Boolean algebra.